src/style.scss

```diff
@@ -7,6 +7,7 @@
   --bg-color: #1c1b20;
   --text-color: #a09fa5;
   --link-color: #faf9fc;
+  --tooltip-color: #a09fa4;
```

Comment:
We don't need to introduce new colors, the color of the tooltip background is exactly the color of borders